docs/source/cluster_management/clis.rst

```diff
@@ -45,4 +45,18 @@ timestamp
 
 Read or recalculate the immutable timestamp. Run ``./bin/atlasdb help timestamp`` for more information.
 
+Offline CLIs
+============
+
+Due to their destructive nature for in-progress transactions, there are two sub-commands of the timestamp CLI which can only be run when AtlasDB is offline. These are:
```

Comment:
Sort of, if you have an existing config and you want it to start an embedded server, you can either specify the `--offline` flag or create a new config without leader/lock/timestamp blocks.